packages/react-tabs/src/components/Pivot/Pivot.styles.ts

```diff
@@ -199,18 +199,27 @@ export const getStyles = (props: IPivotStyleProps): IPivotStyles => {
         [`&[data-is-overflowing='true']`]: {
           display: 'none',
           selectors: {
-            [`~ * .${classNames.overflowMenuButton}`]: {
+            [`~ .${classNames.overflowMenuButton}`]: {
               visibility: 'visible',
+              position: 'relative',
             },
           },
         },
       },
     ],
     overflowMenuButton: [
       classNames.overflowMenuButton,
-      ...getLinkStyles(props, classNames),
       {
-        visibility: 'hidden',
+        // TODO after the extra span wrapper around the button is removed,
+        // remove the 'button&' and 'span&' selectors and combine the styles
+        ['button&']: [...getLinkStyles(props, classNames)],
```

Comment:
```suggestion
        ['button&']: getLinkStyles(props, classNames),
```

Reply:
I had left it like this to make it clearer how to remove this workaround when Button is fixed, but it's probably not necessary :)